### PR TITLE
Routing from internships page to list of candidates 

### PR DESCRIPTION
**Pull request by ValeryiaMIRON**
- Routing from internships page to list of candidates 
- Get data by id in the table

Снимок экрана 2021-11-09 в 9 49 58 PM

https://trello.com/c/HpXS65no/120-fe-get-list-of-candidate-from-internship-pagebutton-list-of-candidates-by-internship-id

### CODE REVIEW

File: src/components/internshipCard/InternshipCard.jsx [hunk 1]
```diff
@@ -47,7 +48,7 @@ export const InternshipCard = ({
       <Button className="internshipCardButton" size="small">
         {getFieldLabel('internships.button.program.info')}
       </Button>
-      <Button size="small">
+      <Button size="small" href={`/candidates/${id}`}>
```
Comment (itsnotpolza) [inline src/components/internshipCard/InternshipCard.jsx:51]: don't use href for navigation

Comment (ValeryiaMIRON) [reply src/components/internshipCard/InternshipCard.jsx:51]: in next PR

File: src/components/header/headerMenu/HeaderMenu.jsx [hunk 2]
```diff
@@ -1,17 +1,21 @@
 import React from 'react';
-import { Button, ButtonGroup } from '@mui/material';
+import { Link } from 'react-router-dom';
+import {
+  Box,
+} from '@mui/material';
```
Comment (itsnotpolza) [inline src/components/header/headerMenu/HeaderMenu.jsx:5]: run format

File: src/components/internshipCard/InternshipCard.jsx [hunk 3]
```diff
@@ -43,13 +44,19 @@ export const InternshipCard = ({
...
+      >
         {getFieldLabel('internships.button.program.info')}
-      </Button>
-      <Button size="small">
+      </Link>
+      <Link
+        to={`/candidates/${id}`}
+        className="internshipCandidatesLink"
+      >
         {getFieldLabel('internships.button.candidates.list')}
```
Comment (itsnotpolza) [inline src/components/internshipCard/InternshipCard.jsx:58]: you can still put button on the inside, just don't provide any onClick

File: src/components/header/headerMenu/HeaderMenu.jsx [hunk 4]
```diff
@@ -1,17 +1,21 @@
...
   return (
-    <ButtonGroup variant="text" aria-label="text button group" color="primary">
-      <Button href="/internships">
-        {getFieldLabel('header.menu.button.internships')}
-      </Button>
-      <Button href="/profile">
-        {getFieldLabel('header.menu.button.profile')}
-      </Button>
-    </ButtonGroup>
+    <Box>
```
Comment (itsnotpolza) [inline src/components/header/headerMenu/HeaderMenu.jsx:11]: you can still use buttons

File: src/components/header/headerMenu/HeaderMenu.jsx (hunk 2 above)
Comment (ValeryiaMIRON) [reply src/components/header/headerMenu/HeaderMenu.jsx]: done

File: src/components/internshipCard/InternshipCard.jsx (hunk 3 above)
Comment (ValeryiaMIRON) [reply src/components/internshipCard/InternshipCard.jsx:48]: ok

File: src/components/header/headerMenu/HeaderMenu.jsx (hunk 4 above)
Comment (ValeryiaMIRON) [reply src/components/header/headerMenu/HeaderMenu.jsx:9]: ok

**Review by beliavskaja — APPROVED** [review]

File: src/pages/home/Home.jsx [hunk 5]
```diff
@@ -22,7 +22,7 @@ function Home() {
               <Route exact path="/internships">
                 <Internships />
               </Route>
-              <Route exact path="/candidates">
```
Comment (itsnotpolza) [inline src/pages/home/Home.jsx:25]: why did you remove exact?

**Review by itsnotpolza — APPROVED** [review]
